app/lib/actions/pulp3/repository/save_artifact.rb

```diff
@@ -26,10 +26,16 @@ def invoke_external_task
               output[:content_unit_href] = existing_content_href
               []
             else
-              output[:pulp_tasks] = [content_backend_service.content_api_create(relative_path: input[:options][:file_name],
-                                                                                repository_id: repository.id,
-                                                                                artifact: artifact_href,
-                                                                                content_type: content_type)]
+              if content_type == 'deb'
+                output[:pulp_tasks] = [content_backend_service.content_api_create(repository_id: repository.id,
+                                                                                  artifact: artifact_href,
+                                                                                  content_type: content_type)]
```

Comment:
Could you make this change in https://github.com/katello/katello/blob/master/app/services/katello/pulp3/deb.rb#L11 and drop the relative_path key from params instead of the generic save_artifact method to avoid the if-else here...

Reply:
Added the changes.